src/routes/supplierRoutes.ts

```diff
@@ -604,20 +613,86 @@ router.post(
   },
 );
 
-// Routes for request outcomes
+router.get(
+  "/received-requests/:requestId/outcome",
+  async (req: Request, res: Response) => {
+    const backLink = req.headers.referer || "/manage-shares/received-requests/";
+    const requestId = req.params.requestId;
 
-router.get("/request-accepted", async (req: Request, res: Response) => {
-  const backLink = req.headers.referer || "/";
-  res.render("../views/supplier/request-accepted.njk", {
-    backLink,
-  });
-});
+    try {
+      const requestDetail = await axios.get(
+        `${URL}/received-requests/${requestId}`,
+        {
+          headers: { Authorization: `Bearer ${req.cookies.jwtToken}` },
+        },
+      );
 
-router.get("/request-rejected", async (req: Request, res: Response) => {
-  const backLink = req.headers.referer || "/";
-  res.render("../views/supplier/request-rejected.njk", {
-    backLink,
-  });
-});
+      if (!requestDetail.data) {
+        return res.status(404).send("Request not found");
+      }
+
+      // Format the received date
+      requestDetail.data.received = formatDate(requestDetail.data.received);
+      requestDetail.data.neededBy = formatDate(requestDetail.data.neededBy);
+      requestDetail.data.decisionDate = formatDate(
+        requestDetail.data.decisionDate,
+      );
+
+      // Format the neededBy date
+      const dateObj = requestDetail.data.sharedata.steps.date.value;
+      requestDetail.data.sharedata.steps.date.formattedValue =
+        formatDateObject(dateObj);
+
+      req.session.acquirerForms = requestDetail.data;
```

Comment:
I'm not sure if we need to be updating the session data here. I think I'd need to look at this again when my changes are merged in too. 
Happy to leave it like this for now - just commenting so I can try and remember